components/script/dom/bindings/global.rs

```diff
@@ -175,10 +176,10 @@ impl<'a> GlobalRef<'a> {
 
     /// `ScriptChan` used to send messages to the event loop of this global's
     /// thread.
-    pub fn user_interaction_task_source(&self) -> Box<ScriptChan + Send> {
+    pub fn user_interaction_task_source(&self) -> Box<TaskSource<UserInteractionTask> + Send> {
```

Comment:
Because workers don't have user interaction. The others are likely used by workers as well as the main thread.